src/dataflow-types/src/types.rs

```diff
@@ -501,6 +502,36 @@ pub mod aws {
     #[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize, Deserialize)]
     pub struct SerdeUri(#[serde(with = "http_serde::uri")] pub Uri);
```

Comment:
I think we should rename this. In the future, we should not be using serde to serialize this at all, we should name it something like `SerializableUri` or something, and eventually we can remove the `Serialize` and `Deserialize` impls and just use the `Codec` impl

Reply:
I'm fine with it, but I think I prefer to do it in a follow-up PR, as Protobuf field names are freely rename-able (as long as the tag number stays the same) and deciding on names is hard.
(for Protobuf, we can also just use `Uri` directly with no problems)

Reply:
sounds good! can you file a small follow-up issue for that?